### PR TITLE
feat: adaptivity first section & header

### PR DESCRIPTION
**Pull request by Nemo-Git-Hub**
[NBO-15](https://trello.com/c/AodWXCOX)

# feat: adaptivity first section & header

### CODE REVIEW

File: website1/src/scss/_components.scss [hunk 1]
```diff
@@ -59,3 +59,41 @@
...
+      width: 208px;
+      height: 30px;
+    }
+    &__shadow {
+      width: 204px;
+      height: 26px;
+      bottom: 14px;
+    }
+  }
+}
```
Comment (asd-ez) [inline website1/src/scss/_components.scss]: Remove explicit height and width assignment.

File: website1/src/scss/sections/_first-page.scss [hunk 2]
```diff
@@ -1,31 +1,88 @@
...
+      &_wide {
+        font-size: 50px;
+      }
+
+      &_default20 {
+        font-size: 14px;
+      }
+    }
+  }
+}
```
Comment (asd-ez) [inline website1/src/scss/sections/_first-page.scss]: If in `{}` only one selector, `{}` are redundant.

File: website1/src/scss/sections/_header.scss [hunk 3]
```diff
@@ -60,3 +63,40 @@
...
+
+@media screen and (max-width: 1280px) {
+  .header {
+    &-nav {
+      &__logo {
+        display: none;
+      }
+    }
+  }
+}
```
Comment (asd-ez) [inline website1/src/scss/sections/_header.scss]: If in `{}` only one selector, `{}` are redundant.

**Review by asd-ez — CHANGES_REQUESTED** [review]
Why did you write adaptive for the header?

File: website1/src/scss/_components.scss [hunk 4]
```diff
@@ -45,17 +91,88 @@
...
+        font-size: 14px;
+        width: 208px;
+        height: 30px;
+      }
+
+      &__shadow {
+        width: 204px;
+        height: 26px;
+        bottom: 14px;
+      }
```
Comment (asd-ez) [inline website1/src/scss/_components.scss]: Do you need explicit width?

File: website1/src/scss/sections/_first-page.scss [hunk 5]
```diff
@@ -1,31 +1,82 @@
...
+
+      &_wide {
+        font-size: 50px;
+      }
+
+      &_default20 {
+        font-size: 14px;
+      }
+    }
+  }
```
Comment (asd-ez) [inline website1/src/scss/sections/_first-page.scss]: Must be sorted by width value.

File: website1/src/scss/_components.scss [hunk 6]
```diff
@@ -21,13 +22,58 @@
...
+//&:active{
+//  transform: translate3d(10px, 10px, 0);
+//  transition: 0.3s;
+//  &::after {
+//    content: "";
+//    transform: translate3d(0, 0, 0);
+//    transition: 2s;
+//  }
+//}
+//}
```
Comment (asd-ez) [inline website1/src/scss/_components.scss:76]: Need to add comments or remove this.

File: website1/dist/index.html [hunk 7]
```diff
@@ -65,7 +65,11 @@
                 Замовити консультацію
               </button>
               <div class="button-active__shadow"></div>
+<!--              <button class="first-page__content-item button-active-second">-->
+<!--                Test Button-->
+<!--              </button>-->
```
Comment (asd-ez) [inline website1/dist/index.html:71]: Need to add comments or remove this.

File: website1/src/scss/_components.scss [hunk 8]
```diff
@@ -45,17 +91,88 @@
...
   }
-}
 
-.title-description {
-  max-width: 626px;
-  width: 100%;
-  line-height: 159%;
-  margin-top: 28px;
+  &-description {
+    max-width: 626px;
```
Comment (asd-ez) [inline website1/src/scss/_components.scss]: It's the universal component, are you sure should be limited by width?
I think if this component describes behavior as `text`, it should contain only styles for text.

File: website1/src/scss/_components.scss (hunk 4 above)
Comment (asd-ez) [inline website1/src/scss/_components.scss]: `button-active` inside `title`?

**Review by asd-ez — CHANGES_REQUESTED** [review]

File: website1/src/scss/_components.scss (hunk 4 above)
Comment (Nemo-Git-Hub) [reply website1/src/scss/_components.scss]: I`m said the button is not ready

Comment (Nemo-Git-Hub) [reply website1/src/scss/_components.scss]: No, of course. Thank You.

File: website1/src/scss/sections/_first-page.scss (hunk 5 above)
Comment (Nemo-Git-Hub) [reply website1/src/scss/sections/_first-page.scss]: Yes, of course

File: website1/dist/index.html (hunk 7 above)
Comment (Nemo-Git-Hub) [reply website1/dist/index.html:71]: created comment

File: website1/src/scss/_components.scss (hunk 6 above)
Comment (Nemo-Git-Hub) [reply website1/src/scss/_components.scss:76]: created comment

File: website1/src/scss/_components.scss (hunk 8 above)
Comment (Nemo-Git-Hub) [reply website1/src/scss/_components.scss]: Yes, it is by design.

File: website1/src/scss/_components.scss (hunk 4 above)
Comment (asd-ez) [reply website1/src/scss/_components.scss]: Write a comment, please. E.g. `FIXME: rebase button`

**Review by asd-ez — APPROVED** [review]

**Pull request merged**